gradle.properties

```diff
@@ -15,4 +15,6 @@
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# tl;dr: Projects are coupled if they change each others config and ours are not
+org.gradle.parallel=true
```

Comment:
i think we should just revert it to being commented out, as it is by default, 1. to reduce PR change noise and 2. in case ever in the future we do need to revisit this, we'll know what was here

we'll always have this PR and our discussions here as reference